packages/ui5-middleware-serveframework/lib/index.js

```diff
@@ -73,16 +74,24 @@ module.exports = async ({ log, options, middlewareUtil }) => {
 
 			// support for coporate proxies
 			const { getProxyForUrl } = await import("proxy-from-env");
-			const proxyUrl = getProxyForUrl(baseUrl);
 			const { HttpsProxyAgent } = await import("https-proxy-agent");
-			const agent = proxyUrl ? new HttpsProxyAgent(proxyUrl) : undefined;
-			effectiveOptions.debug && log.info(`[${baseUrl}] Proxy: ${proxyUrl ? proxyUrl : "n/a"}`);
+			const https = await import("https");
+
+			const proxyUrl = getProxyForUrl(baseUrl);
+			const agentOptions = { rejectUnauthorized: effectiveOptions.strictSSL };
+
+			const agent = proxyUrl ? new HttpsProxyAgent(Object.assign(new URL(proxyUrl), agentOptions)) : new https.Agent(agentOptions);
```

Comment:
`Object.assign`? 

![image](https://github.com/user-attachments/assets/d9062e2a-e915-426d-8fdb-ab12581e3eff)

Reply:
Shouldn't it be: `new HttpsProxyAgent(new URL(proxyUrl), agentOptions)`?

Reply:
true, maybe i was doing something when i was trying some stuff out